.github/workflows/propagate.yml

```diff
@@ -0,0 +1,222 @@
+---
+name: Propagate release templates
+
+# Whenever a PR with changes to the release templates is created in this repo,
+# this workflow will create a branch and an associated PR in each of the
+# repositories defined in the matrix.
+# The branches and PRs will contain the changes to the release templates, so that
+# their contents will be the same as in this repository.
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/ISSUE_TEMPLATE/release.yml
+      - .github/ISSUE_TEMPLATE/cran-release.yml
+  workflow_dispatch:
+
+jobs:
+  propagation:
+    if: github.event.pull_request.draft == false
+    name: Propagate release templates
+    runs-on: ubuntu-latest
+    env:
+      PR_BRANCH_NAME: automated-release-template-propagation
+      DEFAULT_TARGET_BRANCH: main
+      DEFAULT_REVIEWERS: |-
+        KlaudiaBB
+    strategy:
+      matrix:
+        # Run all steps in this job for the following repositories.
+        repos:
+          # All possible options are:
+          # - name: <repository-name-in-current-organization>
+          #   files-to-propagate: '<list-of-files-to-propagate>'
+          #   # If target-branch not specified, DEFAULT_TARGET_BRANCH is used.
+          #   target-branch: <to-which-branch-should-the-files-be-propagated>
+          #   # If reviewers not specified, DEFAULT_REVIEWERS is used.
+          #   reviewers: |-
+          #     <username-1>
+          #     <username-2>
+          #     ...
+          - name: junit-xml-diff
+            files-to-propagate: '.github/ISSUE_TEMPLATE/release.yml .github/ISSUE_TEMPLATE/cran-release.yml'
+          - name: locksmith
+            files-to-propagate: '.github/ISSUE_TEMPLATE/release.yml .github/ISSUE_TEMPLATE/cran-release.yml'
+          - name: scribe
+            files-to-propagate: '.github/ISSUE_TEMPLATE/release.yml .github/ISSUE_TEMPLATE/cran-release.yml'
+
+
+          # - name: hermes
```

Comment:
@KlaudiaBB Can you please verify if this is correct? (For each repository, which of release and cran-release templates should be propagated.)